app/assets/stylesheets/layout_components/callouts/quote-callout.scss

```diff
@@ -1,7 +1,56 @@
 .quote-callout {
-  &__content {
-    @include absolute-center;
+  &__flex {
```

Comment:
Could we name this .quote-callout__container, or something else more descriptive than flex?